kernel_hardening_checker/checks.py

```diff
@@ -265,7 +269,7 @@ def add_kconfig_checks(l, arch):
     l += [KconfigCheck('cut_attack_surface', 'kspp', 'MODIFY_LDT_SYSCALL', 'is not set')]
     l += [KconfigCheck('cut_attack_surface', 'kspp', 'OABI_COMPAT', 'is not set')]
     l += [KconfigCheck('cut_attack_surface', 'kspp', 'X86_MSR', 'is not set')] # refers to LOCKDOWN
-    l += [modules_not_set]
+    l += [OR(modules_not_set, is_android)] # Android requires kernel modules
```

Comment:
Yes, I've found this in the documentation: https://source.android.com/docs/core/architecture/kernel/loadable-kernel-modules